pkg/apis/aws/types_worker.go

```diff
@@ -34,6 +36,12 @@ type WorkerConfig struct {
 	IAMInstanceProfile *IAMInstanceProfile
 }
 
+// NodeTemplate contains information about the expected node properties.
+type NodeTemplate struct {
+	// Capacity represents the expected Node capacity with CPU, GPU and Memory specifications.
+	Capacity corev1.ResourceList
+}
```

Comment:
This can be removed as we can use the NodeTemplate struct defined in g/g

Reply:
Good catch.